etc/logrotate.d/bynar

```diff
@@ -0,0 +1,11 @@
+/var/log/bynar*.log {
+    # Keep 1 week worth of rotated logs
+    rotate 7
```

Comment:
Since we don't have any kinds of alerting and such setup when bynar fails/crashes, at this point I think it may be worthwhile to keep the logs around for a month.

Reply:
Yeah no problem.  The logs are tiny so that shouldn't be an issue